readme.txt

```diff
@@ -62,6 +63,8 @@ It's relatively stable. This plugin has been in the repository since August, 201
 
 = 1.0.1 =
```

Comment:
@marksabbath Do you think the next release should be 1.1.0, as it adds support for two new SEO plugins? Or are we happy with 1.0.1?

Reply:
Good point @nickcernis since there are 2 new features, it does make more sense to me updating it to 1.1.0. @nathanrice do you have any objection on bumping it to 1.1.0 instead of 1.0.1?

Reply:
Not at all!

Reply:
Perfect! @nickcernis I've made a new commit bumping the version to 1.1.0, also, the tested up is 5.0.3.

Reply:
Thanks, @marksabbath — looks good!